migrations/admin_events.go

```diff
@@ -0,0 +1,151 @@
+package migrations
+
+import (
+	"fmt"
+
+	"github.com/mongodb/anser"
+	"github.com/mongodb/anser/db"
+	"github.com/mongodb/anser/model"
+	"github.com/pkg/errors"
+	"gopkg.in/mgo.v2/bson"
+)
+
+const (
+	eventCollection       = "event_log"
+	adminDataType         = "ADMIN"
+	eventTypeKey          = "e_type"
+	eventTypeBanner       = "BANNER_CHANGED"
+	eventTypeTheme        = "THEME_CHANGED"
+	eventTypeServiceFlags = "SERVICE_FLAGS_CHANGED"
+	eventTypeValueChanged = "CONFIG_VALUE_CHANGED"
+)
+
+type eventDataOld struct {
+	ResourceType string            `bson:"r_type"`
+	User         string            `bson:"user"`
+	OldVal       string            `bson:"old_val"`
+	NewVal       string            `bson:"new_val"`
+	OldFlags     eventServiceFlags `bson:"old_flags"`
```

Comment:
You can replace eventServiceFlags with db.Document{}, and delete the eventServiceFlags struct.

Reply:
done